tests/e2e/scenarios/test_reborn_webui_v2_legacy_dom_resource_limits.py

```
@@ -0,0 +1,589 @@
+        composer = page.locator(SEL_V2["chat_composer"])
+        await composer.fill("new message while history page is full")
+        await composer.press("Enter")
+
+        await expect(bubbles).to_have_count(51, timeout=5000)
+        await expect(page.locator(SEL_V2["msg_user"]).last).to_contain_text(
+            "new message while history page is full",
+            timeout=5000,
+        )
+        assert sent_messages and sent_messages[-1]["content"] == (
+            "new message while history page is full"
+        )
```

Ruff (0.15.18)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)